### ISSUE TITLE
Add more non-Latin fonts

### ISSUE THREAD

**Issue by msikma** [ikreymer/netcapsule#16]
Out of all the old browsers that I've tried, only Netscape 4.79 for Linux seemed to be able to display Japanese characters. The others don't seem to have the fonts. Here's an example, using IE 5.1.7 on System 7. This is after setting the document to the correct encoding, which is either EUC or Shift-JIS for Japanese.

![netcapsule_jpn](https://cloud.githubusercontent.com/assets/1507745/11640671/15caf26a-9d34-11e5-845c-6918df3bcab2.png)

It's been a long time since I installed System 7. I remember that a lot of languages were available on the CD for Mac OS 9 (I could provide them) but they might not have been readily available on System 7.

(A small corollary to this is that the pages don't open with the expected character set by default, but that would seem to be impossible to fix unless you start providing images with a system language that isn't English, since there is neither auto-detection nor Unicode.)

**Comment by ikreymer** [comment]
Yes, thanks for reminding me.. I do want to add more fonts but didn't have time before the initial release..
For the Windows based browsers, I was planning to use https://en.wikipedia.org/wiki/Arial_Unicode_MS
I will check to see what's available on System 7.5, though I believe TrueType fonts from OS 8/9 should probably work as well..
